tests/network/vmi_networking.go

```diff
@@ -520,7 +520,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 
 			dnsVMI.Spec.DNSPolicy = "None"
 			dnsVMI.Spec.DNSConfig = &k8sv1.PodDNSConfig{
-				Nameservers: []string{"8.8.8.8", "4.2.2.1"},
+				Nameservers: []string{"8.8.8.8", "04.2.2.01"},
```

Comment:
`04` and `01` are not valid?
The parser is forbidding leading zeros or translating them as octal and only when the range is false it fails?

Reply:
Every octet is parsed as decimal:
04 is parsed as 4, 0127 is parsed as 127.
I've added an example to the PR description as @ormergi had suggested.

Reply:
So this will not fail with the new parsing behavior, because both 1 and 4 are equivalent in value to the octet version.

Reply:
That is right.